docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging_JDBC_Store_for_Messaging_Journal.adoc

```diff
@@ -50,12 +34,28 @@ The name of these tables can be configured with the
 `journal-bindings-table`, `journal-messages-table`,
 `journal-page-store-table, and journal-large-messages-table`.
 
+Please note that the configuration of the underlying pool is something that you need to take care of.
+You  need at least four connections:
+
+ * one for the binding
+ * one for the messages journal
+ * one for the lease lock (if you use HA)
+ * one for the node manager shared state (if you use HA)
+
+So you should define a `min-pool-size` of 4 for the pool. +
+But one fact that you need to take into account is that paging and large messages can use an unbounded number of threads.
+The size of the pool should be defined according to the amount of concurrent threads that perform page/large message streaming operations. There is no defined rule for this as there is no 1-1 relation between the number of threads and the number of connections. +
+You really need to tailor your configuration according to your needs and test it in your environment following the <<DataSource>> subsystem documentation.
```

Comment:
Ok, there is no 1-1 relation, but we should elaborate more about it. Documentation should contains details about options that we have (link to datasource doc could be used, if something in datasource doc is missing, it could be added in this PR). Documentation should contains recommendation and tips how to set datasource settings correctly.

`You really need to tailor your configuration according to your needs`

This is obvious fact, but documentation should provide details how to do this.

Reply:
There is a link to the DataSource subsystem configuration documentation (or am i mistaken in how to use << >>) ?
As it is written it depends on your usage, your database and how many messages and their type (large or not)  you are processing.

Reply:
> There is a link to the DataSource subsystem configuration documentation (or am i mistaken in how to use << >>) ?

Sorry, you are right, I missed the link. Anyway, you [mentioned some details about timeout, etc.](https://github.com/wildfly/wildfly/pull/13965#discussion_r572024822), which I don't see in that datsource documentation.

>  it depends on your usage, your database and how many messages and their type (large or not) you are processing.

Yes, and the details about this "dependency" needs to be elaborated in the doc. As you said, this RFE is mainly about the documentation and so far the documentation doesn't seems to be providing some insight on details about related configuration.

Reply:
Ah, looking to latest update and it seems that you already mentioned details about the timeout. But my previous comment is still valid for `the time to return the connection to the pool or renew it`

Reply:
Should be better now

Reply:
Thanks, current doc in this PR LGTM, I would just wait for AD approval before my final approve of this PR.

Reply:
AD is approved, so I'm ok with this doc now as well